### PR TITLE
rails5

### PR DESCRIPTION
**Pull request by segawa-sh**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -49,16 +47,15 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
-  # config.log_tags = [ :subdomain, :uuid ]
-
-  # Use a different logger for distributed setups.
-  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
+  config.log_tags = [ :request_id ]
```
Comment (segawa-sh) [inline config/environments/production.rb]: **[rubocop]**
- Style/SpaceInsideBrackets: Space inside square brackets detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-spaces-braces)

File: config/environments/production.rb [hunk 2]
```diff
@@ -74,6 +71,16 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
+  # Use a different logger for distributed setups.
+  # require 'syslog/logger'
+  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
+
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
```
Comment (segawa-sh) [inline config/environments/production.rb]: **[rubocop]**
- Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: config/puma.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Puma can serve each request in a thread from an internal thread pool.
+# The `threads` method setting takes two numbers a minimum and maximum.
+# Any libraries that use thread pools should be configured to match
+# the maximum value specified for Puma. Default is set to 5 threads for minimum
+# and maximum, this matches the default thread size of Active Record.
+#
+threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }.to_i
```
Comment (segawa-sh) [inline config/puma.rb]: **[rubocop]**
- Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: config/puma.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+threads threads_count, threads_count
+
+# Specifies the `port` that Puma will listen on to receive requests, default is 3000.
+#
+port        ENV.fetch("PORT") { 3000 }
```
Comment (segawa-sh) [inline config/puma.rb]: **[rubocop]**
- Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: config/puma.rb [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+# Specifies the `environment` that Puma will run in.
+#
+environment ENV.fetch("RAILS_ENV") { "development" }
```
Comment (segawa-sh) [inline config/puma.rb]: **[rubocop]**
- Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: app/policies/application_policy.rb [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    scope.where(:id => record.id).exists?
```
Comment (segawa-sh) [inline app/policies/application_policy.rb]: **[rubocop]**
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

**Comment by tektoh** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[Auto Correct by SideCI](https://sideci.com/github_repositories/47532393/issues/pull_request/16/diff/c61aeb5d89f3a6095587b8cc9ae19d7079da5526?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Pull request merged**